docs/UserGuide.md

```diff
@@ -251,29 +306,35 @@ Deletes all completed assignments from TA<sup>2</sup>.
 
 Format: `clean`
 
-### Clearing all entries : `clear`
+### Clearing all entries: `clear`
 
 Clears all entries from TA<sup>2</sup>.
 
 Format: `clear`
 
-### Undoing the last command : `undo`
+### Undoing a command: `undo`
 
 Undoes the last command entered.
-* Undo all kinds of command except for `undo` and `redo`.
-* At the start of program, nothing can be undone.
 
-Format: `undo` 
+Format: `undo`
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
I think the format for Caution and the following lines don't show up correctly in the preview page. Maybe you need to add an extra spacing or <br>

Reply:
good catch,done